docs/How-to-test-the-application.md

```diff
@@ -0,0 +1,59 @@
+# How to test App:
+- Using Physical device and apk
+- Using Emulator and Apk
+- Directly Running in Android Studio ( Recommended )
+
+
+## For option one or two:
+
+Every time a new commit is merged into the `develop` branch an _apk_ is released in the `apk` branch. To Navigate to `apk` Branch see here: 
```

Comment:
Can you correct the grammar in line number 9? Also the B in branch should be small.

Reply:
LGTM passes grammar check everywhere your suggestions?